src/main/java/com/paneedah/mwc/wearables/Backpacks.java

```diff
@@ -43,27 +43,27 @@ public class Backpacks {
 
     public static void init(Object mod, CompatibleFmlPreInitializationEvent event) {
         
-        TacticalPushPack = new ItemStorage.Builder()
-        .withName("tactical_push_backpack")
-        .withSize(8)
-        .withGuiTextureWidth(256)
-        .withValidItemPredicate(item -> !(item instanceof Weapon))
-        .withTab(CreativeTabs.MISC)
-        .withModel(new com.paneedah.mwc.models.TacticalPushPack())
-        .withModelTextureName("black_tru_spec_cordura_backpack.png")
-        .withGuiTextureName("tactical_push_pack.png")
-        .withCustomEquippedPositioning((player, stack) -> {
-            GL11.glScalef(0.8f, 0.8f, 0.8f);
-            GL11.glTranslatef(0f, 0f, 0f);
-            GL11.glRotatef(0F, 0f, 0f, 1f);
-        })
-        .withInventoryPositioning(stack -> {
-            GL11.glScalef(1.3f, 1.3f, 1.3f);
-            GL11.glTranslatef(0.2f, -0.55f, 0f);
-            GL11.glRotatef(23F, 0f, 1f, 0f);
-            GL11.glRotatef(20F, 0f, 0f, 1f);
-        })
-        .build(ModernWarfareMod.MOD_CONTEXT);
+//        TacticalPushPack = new ItemStorage.Builder()
+//        .withName("tactical_push_backpack")
+//        .withSize(8)
+//        .withGuiTextureWidth(256)
+//        .withValidItemPredicate(item -> !(item instanceof Weapon))
+//        .withTab(CreativeTabs.MISC)
+//        .withModel(new com.paneedah.mwc.models.TacticalPushPack())
+//        .withModelTextureName("black_tru_spec_cordura_backpack.png")
+//        .withGuiTextureName("tactical_push_pack.png")
+//        .withCustomEquippedPositioning((player, stack) -> {
+//            GL11.glScalef(0.8f, 0.8f, 0.8f);
+//            GL11.glTranslatef(0f, 0f, 0f);
+//            GL11.glRotatef(0F, 0f, 0f, 1f);
+//        })
+//        .withInventoryPositioning(stack -> {
+//            GL11.glScalef(1.3f, 1.3f, 1.3f);
+//            GL11.glTranslatef(0.2f, -0.55f, 0f);
+//            GL11.glRotatef(23F, 0f, 1f, 0f);
+//            GL11.glRotatef(20F, 0f, 0f, 1f);
+//        })
+//        .build(ModernWarfareMod.MOD_CONTEXT);
```

Comment:
Same, I may be mistaken but this is not contagion related.

Reply:
this should've been with the "MASTER" class push instead but made a mistake. but It still needs to get removed

Reply:
It was supposed to be in Master Branch but I messed up but It's still broken either way.